### ISSUE TITLE
Retrieve all storage

### ISSUE THREAD

**Issue by fatoldsun00** [ammarahm-ed/react-native-mmkv-storage#59]
Hi it's possible to get all storage ? I wish hydrate a store on load, so there are a function like .get() for retrieve all key in a loader ?

Thanks

**Comment by ammarahm-ed** [comment]
Yup there is, check the docs, indexer part

**Comment by fatoldsun00** [comment]
In so stupid, i readed instance indexer section really to fast !

getKeys
Get all the keys in the storage for all stored items.

Sorry, sorry, sorry for wasted time and thanks for answering

**Comment by aureosouza** [comment]
We noticed `MMKV.indexer.getKeys()` returns everything, including config keys like `stringIndex`and `boolIndex` or the values from withInstanceID. We were looking for an easy way to reset all the regular values not the config, without having to call for each types `indexer.strings.getKeys`, `indexer.booleans.getKeys` etc. We noticed that removing the config keys as well, leads to unnexpected reactive behaviour of useMMKVStorage. So a possible improvement could be a new function `indexer.getRegularKeys()` that would return only the regular keys, but not the config ones.
